.github/workflows/main.yaml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on:
+  # Run every day at 10 pm
+  #schedule:
+  #  # Temporary cron for testing.
+  #  - cron: '*/15 * * * *'
+  
+  push:
+    branches: [ "williamhe/**" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      releaseDate:
+        description: 'The date to check for new releases. Format: yyyy-mm-dd'
+        required: true
+        type: string
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+
+      - name: Run SDK automation
+        run: time dotnet run --project build/tools/Automation/Automation.csproj
+
+        # [Important]
+        # Currently fails silently due to malformed path for Hugo.
+        # Unhandled exception. System.IO.DirectoryNotFoundException: Could not find a part of the path '/home/runner/work/Oryx/Oryx/build\detector/__detectorNugetPackagesVersions.props'.
```

Comment:
Yep the paths are from [this block](https://github.com/microsoft/Oryx/blob/main/build/constants.yaml#L353-L379). Lets just try to change the 3 `directory: build\detector` instances to `directory: build/detector` (forward slash) and see if that works?

Reply:
Changing from `\` to `/` in the constants yaml file has no impact on the location that the file is placed when generateConstants.sh is run, and all projects still build. I was thinking that it would also fix this issue, but are you saying that it doesn't?

Reply:
Sorry Paul I didn't see both of your messages, perhaps the page didn't refresh. But I updated the slashes and it works fine now. Thanks!